templates/mydata_status.html

```diff
@@ -0,0 +1,133 @@
+{% load crispy_forms_tags %}
+
+{% block mydata_page_content %}
+  <div class="tabpanel">
+    <h2 style="margin-top: 1em;">Twój status w edycji {{ current_year }}:</h2>
+    {% if not current_status %}
+      <div style="font-size: 300%;">Nie zgłoszono</div>
+      <div>
+        Nie zgłosiłeś się na żadne warsztaty. Przejdź do zakładki <a href="{% url 'program' current_year.pk %}">Program</a>,
+        zapisz się na interesujące Cię tematy i rozwiąż na nie zadania kwalifikacyjne. Wypełnij także swój
+        <a href="{% url 'mydata_profile' %}">profil</a> i list motywacyjny.
+      </div>
+    {% else %}
+      {% if current_status.type == 'lecturer' %}
+        {% if current_status.status == 'Z' %}
+          <div style="font-size: 300%;"><span class="qualified">Zaakceptowano jako prowadzącego</span></div>
+          <div>
+            Twoje warsztaty zostały zaakceptowane. Czekaj na dalsze informacje.
+          </div>
+        {% elif current_status.status == 'O' %}
+          <div style="font-size: 300%;"><span class="not-qualified">Odrzucono jako prowadzącego</span></div>
+          <div>
+            Twoje warsztaty zostały odrzucone
+          </div>
+        {% elif current_status.status == 'X' %}
+          <div style="font-size: 300%;"><span class="maybe-qualified">Odwołano przyjazd prowadzącego</span></div>
+          <div>
+            Twoje warsztaty były zaakceptowane, ale zostały odwołane
+          </div>
+        {% elif not current_status.status %}
+          <div style="font-size: 300%;"><span class="waiting-qualified">Zgłoszono prowadzącego</span></div>
+          <div>
+            Twoje warsztaty zostały zarejestrowane w systemie. Poczekaj na decyzję organizatorów.
+          </div>
+        {% endif %}
+      {% else %}
+        {% if current_status.status == 'Z' %}
+          <div style="font-size: 300%;"><span class="qualified">Zaakceptowano</span></div>
+          <div>
+            {% if gender == 'F' %}
+              Zostałaś zakwalifikowana na warsztaty. Czekaj na dalsze informacje.
+            {% else %}
+              Zostałeś zakwalifikowany na warsztaty. Czekaj na dalsze informacje.
+            {% endif %}
+          </div>
+        {% elif current_status.status == 'O' %}
+          <div style="font-size: 300%;"><span class="not-qualified">Odrzucono</span></div>
+          <div>
+            Niestety, nie mogliśmy przyjąć Cię na warsztaty w tym roku.
+          </div>
+        {% elif current_status.status == 'X' %}
+          <div style="font-size: 300%;"><span class="maybe-qualified">Odwołano przyjazd</span></div>
+          <div>
+            {% if gender == 'F' %}
+              Zostałaś zakwalifikowana na warsztaty, ale odwołałaś swój przyjazd.
+            {% else %}
+              Zostałeś zakwalifikowany na warsztaty, ale odwołałeś swój przyjazd.
+            {% endif %}
+          </div>
+        {% elif not current_status.status %}
+          {% if not has_cover_letter %}
```

Comment:
We need indication of profile info missing, but I would leave this for forms rework.

Reply:
I know, that's why I didn't touch it for now